pkg/drivers/dqlite/no_dqlite.go

```diff
@@ -11,5 +11,5 @@ import (
 )
 
 func New(ctx context.Context, datasourceName string, connPoolConfig generic.ConnectionPoolConfig) (server.Backend, error) {
-	return nil, fmt.Errorf("dqlite is not support, compile with \"-tags dqlite\"")
+	return nil, fmt.Errorf("this binary is built without dqlite support, compile with \"-tags dqlite\"")
```

Comment:
Since there's no value being passed in for use in the string formatting, `errors.New` is preferred.

Reply:
Sure, I can clean that up while I'm here. I was mostly just irked by the poor grammar.